include/groonga/table.h

```diff
@@ -709,6 +709,18 @@ grn_table_setoperation(grn_ctx *ctx,
                        grn_obj *table2,
                        grn_obj *res,
                        grn_operator op);
+/**
+ * \brief Delete duplicate records in `table1` and `table2`.
+ *
+ * \param ctx The context object.
+ * \param table1 The table.
+ * \param table2 The table.
+ * \param res1 Specify `table1`.
+ * \param res2 Specify `table2`.
+ *
+ * \return \ref GRN_SUCCESS on success, \ref GRN_INVALID_ARGUMENT on
+ *         `table1 != res1 || table2 != res2`.
+ */
 GRN_API grn_rc
 grn_table_difference(
```

Comment:
Is the grn_table_difference function no longer in use? If that's the case, should we consider adding a deprecation notice?

Reply:
We don't need to do it so far.

We may want to do it when we add `GRN_OP_DIFFERENCE` and add support for it in `grn_table_setoperation()`. (But in-place difference can't be done with `grn_table_setoperation()` interface...)

Reply:
I got it, thank you!